content/en/smart-contracts/fundamentals/the-filecoin-virtual-machine/index.md

```diff
@@ -23,222 +23,44 @@ aliases:
     - "/developers/smart-contracts/about/use-cases/"
 ---
 
-## Features
+## Introduction
 
-The Filecoin virtual machine (FVM) was created to enable developers to build new use cases on top of the Filecoin network. Data access control, data DAOs, perpetual storage, collateral leasing, and Ethereum-compatible fungible and non-fungible tokens are just some of the use-cases for the FVM.
+Filecoin’s storage and retrieval capabilities can be thought of as the base layer of the Filecoin blockchain, and FVM can be thought of as a layer on top of Filecoin enabling programmatic execution of storage and retrieval.  
```

Comment:
> can be thought of

Using this phrase twice in a row isn't ideal.

Reply:
Ok, this is good, but let's keep the list of use cases.  Should read: 

"Filecoin’s storage and retrieval capabilities can be thought of as the base layer of the Filecoin blockchain, and FVM can be thought of as a layer on top of Filecoin enabling programmatic execution of storage and retrieval.  Data access control, data DAOs, perpetual storage, collateral leasing, and Ethereum-compatible fungible and non-fungible tokens are just some of the use-cases for the FVM."